arrow-core-data/src/main/kotlin/arrow/typeclasses/Alternative.kt

```diff
@@ -64,8 +63,17 @@ interface Alternative<F> : Applicative<F>, MonoidK<F> {
    * @receiver computation to execute
    * @return Option with either the result or none in case it failed
    */
+  @Deprecated("Please use nullable")
```

Comment:
I find `ReplaceWith` works best when the function signature doesn't change, or can be corrected automatically. Here is most cases that can't be done, so I didn't do it.

Reply:
Yeah, I guess that makes sense, specially because we don't have all the Option operations available for nullables... If we added those then we could do a seamless conversion.
I'll add a note on the ticket for fold